lib/cosmos/api/cmd_api.rb

```diff
@@ -182,7 +192,7 @@ def send_raw(interface_name, data, scope: $cosmos_scope, token: $cosmos_token)
     # @return [String] last command buffer packet
     def get_cmd_buffer(target_name, command_name, scope: $cosmos_scope, token: $cosmos_token)
       authorize(permission: 'cmd_info', target_name: target_name, packet_name: command_name, scope: scope, token: token)
-      Store.instance.cmd_packet_exist?(target_name, command_name, scope: scope)
+      TargetModel.packet_exist(target_name, command_name, type: :CMD, scope: scope)
```

Comment:
Moved some of the Store stuff into models. I think TargetModel is where we should check for packets existence as it creates them.